diff -rupN --no-dereference gdal-3.2.2-fedora/m4/geos.m4 gdal-3.2.2-fedora-new/m4/geos.m4
--- gdal-3.2.2-fedora/m4/geos.m4	2021-03-05 12:39:54.000000000 +0100
+++ gdal-3.2.2-fedora-new/m4/geos.m4	2021-03-24 13:44:41.905428811 +0100
@@ -129,6 +129,8 @@ AC_DEFUN([GEOS_INIT],[
       LIBS=${GEOS_LIBS}
       ax_save_CFLAGS="${CFLAGS}"
       CFLAGS="${GEOS_CFLAGS}"
+      ax_save_LDFLAGS="${LDFLAGS}"
+      LDFLAGS=""
 
       AC_CHECK_LIB([geos_c],
         [GEOSversion],
@@ -143,6 +145,7 @@ AC_DEFUN([GEOS_INIT],[
 
       CFLAGS="${ax_save_CFLAGS}"
       LIBS="${ax_save_LIBS}"
+      LDFLAGS="${ax_save_LDFLAGS}"
 
     fi
 
diff -rupN --no-dereference gdal-3.2.2-fedora/m4/sfcgal.m4 gdal-3.2.2-fedora-new/m4/sfcgal.m4
--- gdal-3.2.2-fedora/m4/sfcgal.m4	2021-03-05 12:39:52.000000000 +0100
+++ gdal-3.2.2-fedora-new/m4/sfcgal.m4	2021-03-24 13:44:41.905428811 +0100
@@ -128,6 +128,8 @@ AC_DEFUN([SFCGAL_INIT],[
       LIBS=${SFCGAL_LIBS}
       ax_save_CFLAGS="${CFLAGS}"
       CFLAGS="${SFCGAL_CFLAGS}"
+      ax_save_LDFLAGS="${LDFLAGS}"
+      LDFLAGS=""
 
       AC_CHECK_LIB([SFCGAL],
         [sfcgal_version],
@@ -143,6 +145,7 @@ AC_DEFUN([SFCGAL_INIT],[
 
       CFLAGS="${ax_save_CFLAGS}"
       LIBS="${ax_save_LIBS}"
+      LDFLAGS="${ax_save_LDFLAGS}"
 
     fi
 
