diff -rupN --no-dereference gdal-3.1.4-fedora/swig/java/java.opt gdal-3.1.4-fedora-new/swig/java/java.opt
--- gdal-3.1.4-fedora/swig/java/java.opt	2020-10-20 11:47:08.000000000 +0200
+++ gdal-3.1.4-fedora-new/swig/java/java.opt	2020-11-02 11:56:48.341314610 +0100
@@ -7,6 +7,6 @@ JAVADOC=$(JAVA_HOME)/bin/javadoc
 JAVAC=$(JAVA_HOME)/bin/javac
 JAVA=$(JAVA_HOME)/bin/java
 JAR=$(JAVA_HOME)/bin/jar
-JAVA_INCLUDE=-I$(JAVA_HOME)/include -I$(JAVA_HOME)/include/linux
+JAVA_INCLUDE=-I/usr/lib/jvm/java/include/ -I/usr/lib/jvm/java/include/linux
 
 
