diff -rupN gdal-2.2.3-fedora/configure.ac gdal-2.2.3-fedora-new/configure.ac
--- gdal-2.2.3-fedora/configure.ac	2017-11-20 11:45:32.000000000 +0100
+++ gdal-2.2.3-fedora-new/configure.ac	2018-02-13 09:44:58.214951534 +0100
@@ -2089,6 +2089,8 @@ dnl Fedora has hdf static libraries in /
   if test "$HDF_LIB_DIR" != "" ; then
     LIBS="-L$HDF_LIB_DIR $LIBS"
   fi
+  EXTRA_INCLUDES="-I/usr/include/tirpc/ $EXTRA_INCLUDES"
+  LIBS="-ltirpc $LIBS"
 
 dnl Debian supplies the HDF4 library which does not conflict with NetCDF.
 dnl Test for Debian flavor first. Hint: install the libhdf4-alt-dev package.
