diff -rupN --no-dereference gdal-3.6.0-fedora/port/vsipreload.cpp gdal-3.6.0-fedora-new/port/vsipreload.cpp
--- gdal-3.6.0-fedora/port/vsipreload.cpp	2022-11-06 21:09:21.000000000 +0100
+++ gdal-3.6.0-fedora-new/port/vsipreload.cpp	2022-11-11 18:33:22.896005450 +0100
@@ -55,6 +55,17 @@
 #undef _FORTIFY_SOURCE
 #endif
 
+
+/*
+ * We need to export open* etc., but _FORTIFY_SOURCE defines conflicting
+ * always_inline versions. Disable _FORTIFY_SOURCE for this file, so we
+ * can define our overrides.
+ */
+#ifdef _FORTIFY_SOURCE
+#undef _FORTIFY_SOURCE
+#endif
+
+
 #define _GNU_SOURCE 1
 #define _LARGEFILE64_SOURCE 1
 #include <stdio.h>
