diff -rupN --no-dereference gdal-3.4.2-fedora/doc/Makefile gdal-3.4.2-fedora-new/doc/Makefile
--- gdal-3.4.2-fedora/doc/Makefile	2022-03-08 11:10:52.000000000 +0100
+++ gdal-3.4.2-fedora-new/doc/Makefile	2022-03-14 23:50:44.164160839 +0100
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    ?= --keep-going -j auto -W
+SPHINXOPTS    ?= --keep-going
 SPHINXBUILD   = sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
