diff -rupN --no-dereference gdal-3.3.1-fedora/doc/Makefile gdal-3.3.1-fedora-new/doc/Makefile
--- gdal-3.3.1-fedora/doc/Makefile	2021-06-28 12:57:57.000000000 +0200
+++ gdal-3.3.1-fedora-new/doc/Makefile	2021-07-05 16:04:47.871901605 +0200
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    ?= --keep-going -j auto -W
+SPHINXOPTS    ?= --keep-going
 SPHINXBUILD   = sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
