diff -rupN --no-dereference gdal-3.4.0-fedora/m4/lib-link.m4 gdal-3.4.0-fedora-new/m4/lib-link.m4
--- gdal-3.4.0-fedora/m4/lib-link.m4	2021-11-04 12:57:06.000000000 +0100
+++ gdal-3.4.0-fedora-new/m4/lib-link.m4	2021-11-09 11:58:38.585300818 +0100
@@ -108,8 +108,6 @@ dnl   acl_hardcode_direct,
 dnl   acl_hardcode_minus_L.
 AC_DEFUN([AC_LIB_RPATH],
 [
-  dnl Tell automake >= 1.10 to complain if config.rpath is missing.
-  m4_ifdef([AC_REQUIRE_AUX_FILE], [AC_REQUIRE_AUX_FILE([config.rpath])])
   AC_REQUIRE([AC_PROG_CC])                dnl we use $CC, $GCC, $LDFLAGS
   AC_REQUIRE([AC_LIB_PROG_LD])            dnl we use $LD, $with_gnu_ld
   AC_REQUIRE([AC_CANONICAL_HOST])         dnl we use $host
