diff -rupN --no-dereference gdal-3.2.2-fedora/swig/java/java.opt gdal-3.2.2-fedora-new/swig/java/java.opt
--- gdal-3.2.2-fedora/swig/java/java.opt	2021-03-05 12:38:35.000000000 +0100
+++ gdal-3.2.2-fedora-new/swig/java/java.opt	2021-03-24 13:44:40.268428769 +0100
@@ -7,6 +7,6 @@ JAVADOC=$(JAVA_HOME)/bin/javadoc
 JAVAC=$(JAVA_HOME)/bin/javac
 JAVA=$(JAVA_HOME)/bin/java
 JAR=$(JAVA_HOME)/bin/jar
-JAVA_INCLUDE=-I$(JAVA_HOME)/include -I$(JAVA_HOME)/include/linux
+JAVA_INCLUDE=-I/usr/lib/jvm/java/include/ -I/usr/lib/jvm/java/include/linux
 
 
