diff -rupN --no-dereference gdal-3.1.2-fedora/doc/Makefile gdal-3.1.2-fedora-new/doc/Makefile
--- gdal-3.1.2-fedora/doc/Makefile	2020-07-07 13:43:38.000000000 +0200
+++ gdal-3.1.2-fedora-new/doc/Makefile	2020-07-07 15:17:33.445889094 +0200
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = --keep-going -j auto -W
+SPHINXOPTS    = --keep-going
 SPHINXBUILD   = sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
