diff -rupN --no-dereference gdal-3.3.2-fedora/doc/Makefile gdal-3.3.2-fedora-new/doc/Makefile
--- gdal-3.3.2-fedora/doc/Makefile	2021-09-01 11:48:26.000000000 +0200
+++ gdal-3.3.2-fedora-new/doc/Makefile	2021-09-07 12:51:18.095931201 +0200
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    ?= --keep-going -j auto -W
+SPHINXOPTS    ?= --keep-going
 SPHINXBUILD   = sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
