diff -rupN --no-dereference gdal-3.1.3-fedora/frmts/jpeg2000/jpeg2000_vsil_io.cpp gdal-3.1.3-fedora-new/frmts/jpeg2000/jpeg2000_vsil_io.cpp
--- gdal-3.1.3-fedora/frmts/jpeg2000/jpeg2000_vsil_io.cpp	2020-09-01 10:39:25.000000000 +0200
+++ gdal-3.1.3-fedora-new/frmts/jpeg2000/jpeg2000_vsil_io.cpp	2020-09-07 14:41:33.957532225 +0200
@@ -96,7 +96,7 @@ typedef struct {
 
 // PRIjas_seqent macro is defined since Jasper 2.0.17
 
-#ifndef PRIjas_seqent
+#if 0
 static int JPEG2000_VSIL_read(jas_stream_obj_t *obj, char *buf, int cnt)
 #else
 static int JPEG2000_VSIL_read(jas_stream_obj_t *obj, char *buf, unsigned cnt)
@@ -106,7 +106,7 @@ static int JPEG2000_VSIL_read(jas_stream
     return static_cast<int>(VSIFReadL(buf, 1, cnt, fileobj->fp));
 }
 
-#ifndef PRIjas_seqent
+#if 0
 static int JPEG2000_VSIL_write(jas_stream_obj_t *obj, char *buf, int cnt)
 #else
 static int JPEG2000_VSIL_write(jas_stream_obj_t *obj, char *buf, unsigned int cnt)
