diff -Nru gdal-1.4.2-fedora/frmts/formats_list.html gdal-1.4.2-fedora-patched/frmts/formats_list.html
--- gdal-1.4.2-fedora/frmts/formats_list.html	2007-06-27 20:47:46.000000000 +0200
+++ gdal-1.4.2-fedora-patched/frmts/formats_list.html	2007-07-13 14:22:08.000000000 +0200
@@ -178,13 +178,6 @@
 </td><td> 2GiB
 </td></tr>
 
-<tr><td> <a href="frmt_hfa.html">Erdas Imagine (.img)</a>
-</td><td> HFA
-</td><td> Yes
-</td><td> Yes
-</td><td> No limits<sup><a href="#footnote2">2</a></sup>
-</td></tr>
-
 <tr><td> <a href="frmt_mff2.html">Vexcel MFF2</a>
 </td><td> HKV
 </td><td> Yes
@@ -262,13 +255,6 @@
 </td><td> --
 </td></tr>
 
-<tr><td> <a href="frmt_various.html#LAN">Erdas 7.x .LAN and .GIS</a>
-</td><td> LAN
-</td><td> No
-</td><td> Yes
-</td><td> 2GB
-</td></tr>
-
 <tr><td> <a href="frmt_leveller.html">Daylon Leveller Heightfield</a>
 </td><td> Leveller
 </td><td> Yes
@@ -440,12 +426,6 @@
 </p>
 
 <p>
-<a name="#footnote2"><sup>2</sup>ERDAS Imagine has different file format for
-	large files, where 32-bit pointers cannot be used. Look for details
-	<a href="frmt_hfa.html">here</a>.
-</p>
-
-<p>
 $Id: gdal-buildfix.patch,v 1.3 2007/07/25 13:28:08 cbalint Exp $
 </p>
 
diff -Nru gdal-1.4.2-fedora/frmts/gdalallregister.cpp gdal-1.4.2-fedora-patched/frmts/gdalallregister.cpp
--- gdal-1.4.2-fedora/frmts/gdalallregister.cpp	2007-06-27 20:47:46.000000000 +0200
+++ gdal-1.4.2-fedora-patched/frmts/gdalallregister.cpp	2007-07-13 14:22:08.000000000 +0200
@@ -88,10 +88,6 @@
     GDALRegister_NITF();
 #endif
 
-#ifdef FRMT_hfa
-    GDALRegister_HFA();
-#endif
-    
 #ifdef FRMT_ceos2
     GDALRegister_SAR_CEOS();
 #endif
diff -Nru gdal-1.4.2-fedora/frmts/makefile.vc gdal-1.4.2-fedora-patched/frmts/makefile.vc
--- gdal-1.4.2-fedora/frmts/makefile.vc	2007-06-27 20:47:46.000000000 +0200
+++ gdal-1.4.2-fedora-patched/frmts/makefile.vc	2007-07-13 14:22:08.000000000 +0200
@@ -1,7 +1,7 @@
 
 GDAL_ROOT = ..
 
-EXTRAFLAGS =	-DFRMT_ceos -DFRMT_aigrid -DFRMT_elas -DFRMT_hfa -DFRMT_gtiff\
+EXTRAFLAGS =	-DFRMT_ceos -DFRMT_aigrid -DFRMT_elas -DFRMT_gtiff\
 		-DFRMT_sdts -DFRMT_raw -DFRMT_gxf -DFRMT_ceos2 -DFRMT_png \
 		-DFRMT_dted -DFRMT_mem -DFRMT_jdem -DFRMT_gif \
 		-DFRMT_envisat -DFRMT_aaigrid -DFRMT_usgsdem -DFRMT_l1b \
diff -Nru gdal-1.4.2-fedora/gcore/gdal_frmts.h gdal-1.4.2-fedora-patched/gcore/gdal_frmts.h
--- gdal-1.4.2-fedora/gcore/gdal_frmts.h.orig     2007-07-25 09:44:18.000000000 +0200
+++ gdal-1.4.2-fedora-patched/gcore/gdal_frmts.h  2007-07-25 09:44:28.000000000 +0200
@@ -37,7 +37,6 @@
 void CPL_DLL GDALRegister_GTiff(void);
 void CPL_DLL GDALRegister_GXF(void);
 void CPL_DLL GDALRegister_OGDI(void);
-void CPL_DLL GDALRegister_HFA(void);
 void CPL_DLL GDALRegister_AAIGrid(void);
 void CPL_DLL GDALRegister_AIGrid(void);
 void CPL_DLL GDALRegister_AIGrid2(void);
diff -Nru gdal-1.4.2-fedora/GDALmake.opt.in gdal-1.4.2-fedora-patched/GDALmake.opt.in
--- gdal-1.4.2-fedora/GDALmake.opt.in	2007-06-27 20:48:17.000000000 +0200
+++ gdal-1.4.2-fedora-patched/GDALmake.opt.in	2007-07-13 14:22:08.000000000 +0200
@@ -312,7 +312,7 @@
 #	and their uppercase form should be the format portion of the
 #	format registration entry point.  eg. gdb -> GDALRegister_GDB().
 #
-GDAL_FORMATS = 	gxf gtiff hfa aigrid aaigrid ceos ceos2 iso8211 xpm \
+GDAL_FORMATS = 	gxf gtiff aigrid aaigrid ceos ceos2 iso8211 xpm \
 		sdts raw dted mem jdem envisat elas fit vrt usgsdem l1b \
 		nitf bmp pcidsk airsar rs2 ilwis rmf rik leveller sgi \
 		idrisi \
diff -Nru gdal-1.4.2-fedora/ogr/ogrsf_frmts/generic/ogrregisterall.cpp gdal-1.4.2-fedora-patched/ogr/ogrsf_frmts/generic/ogrregisterall.cpp
--- gdal-1.4.2-fedora/ogr/ogrsf_frmts/generic/ogrregisterall.cpp	2007-06-27 20:46:10.000000000 +0200
+++ gdal-1.4.2-fedora-patched/ogr/ogrsf_frmts/generic/ogrregisterall.cpp	2007-07-13 14:22:08.000000000 +0200
@@ -57,9 +57,6 @@
 #ifdef S57_ENABLED
     RegisterOGRS57();
 #endif
-#ifdef DGN_ENABLED
-    RegisterOGRDGN();
-#endif
 #ifdef VRT_ENABLED
     RegisterOGRVRT();
 #endif
@@ -81,10 +78,6 @@
 #ifdef KML_ENABLED
     RegisterOGRKML();
 #endif
-#ifdef ILI_ENABLED
-    RegisterOGRILI1();
-    RegisterOGRILI2();
-#endif
 #ifdef SQLITE_ENABLED
     RegisterOGRSQLite();
 #endif
diff -Nru gdal-1.4.2-fedora/ogr/ogrsf_frmts/GNUmakefile gdal-1.4.2-fedora-patched/ogr/ogrsf_frmts/GNUmakefile
--- gdal-1.4.2-fedora/ogr/ogrsf_frmts/GNUmakefile	2007-06-27 20:46:19.000000000 +0200
+++ gdal-1.4.2-fedora-patched/ogr/ogrsf_frmts/GNUmakefile	2007-07-13 14:22:08.000000000 +0200
@@ -1,7 +1,7 @@
 
 include ../../GDALmake.opt
 
-SUBDIRS-yes	:= generic shape ntf sdts tiger s57 mitab dgn \
+SUBDIRS-yes	:= generic shape ntf sdts tiger s57 mitab \
 				gml kml avc rec mem vrt csv
 SUBDIRS-$(HAVE_OGDI)	+= ogdi
 SUBDIRS-$(HAVE_OCI)	+= oci
@@ -12,7 +12,6 @@
 SUBDIRS-$(ODBC_SETTING)	+= odbc pgeo
 SUBDIRS-$(HAVE_SQLITE)	+= sqlite
 SUBDIRS-$(HAVE_GRASS)	+= grass
-SUBDIRS-$(HAVE_XERCES)	+= ili
 SUBDIRS-$(HAVE_DWGDIRECT) += dxfdwg
 SUBDIRS-$(HAVE_SDE)     += sde
 SUBDIRS-$(HAVE_IDB)	+= idb
diff -Nru gdal-1.4.2-fedora/ogr/ogrsf_frmts/Makefile gdal-1.4.2-fedora-patched/ogr/ogrsf_frmts/Makefile
--- gdal-1.4.2-fedora/ogr/ogrsf_frmts/Makefile	2007-06-27 20:46:19.000000000 +0200
+++ gdal-1.4.2-fedora-patched/ogr/ogrsf_frmts/Makefile	2007-07-13 14:22:08.000000000 +0200
@@ -1,6 +1,6 @@
 GDAL_ROOT	=	..\..
 
-DIRLIST		=	generic shape ntf sdts tiger s57 dgn mitab gml kml avc \
+DIRLIST		=	generic shape ntf sdts tiger s57 mitab gml kml avc \
 			rec mem vrt csv \
 			$(OGDIDIR) $(FMEDIR) $(OCIDIR) $(PG_DIR) $(DWGDIR) \
 			$(ODBCDIR) $(SQLITE_DIR) $(MYSQL_DIR) $(ILI_DIR) \
@@ -53,11 +53,6 @@
 MYSQL_OBJ = mysql\*.obj
 !ENDIF
 
-!IFDEF ILI_ENABLED
-ILI_DIR = ili
-ILI_OBJ = ili\*.obj ili\iom\*.obj
-!ENDIF
-
 !IFDEF INFORMIXDIR
 IDB_DIR = idb
 IDB_OBJ = idb\*.obj
@@ -71,7 +66,7 @@
 
 	lib /out:ogrsf_frmts.lib generic\*.obj shape\*.obj ntf\*.obj \
 				 sdts\*.obj s57\*.obj tiger\*.obj gml\*.obj \
-				 mitab\*.obj dgn\*.obj avc\*.obj mem\*.obj \
+				 mitab\*.obj avc\*.obj mem\*.obj \
 				 vrt\*.obj csv\*.obj rec\*.obj kml\*.obj \
 				$(OGDIOBJ) $(ODBCOBJ) $(SQLITE_OBJ) \
 				$(FMEOBJ) $(OCIOBJ) $(PG_OBJ) $(MYSQL_OBJ) \
diff -Nru gdal-1.4.2-fedora/ogr/ogrsf_frmts/ogr_formats.html gdal-1.4.2-fedora-patched/ogr/ogrsf_frmts/ogr_formats.html
--- gdal-1.4.2-fedora/ogr/ogrsf_frmts/ogr_formats.html	2007-06-27 20:46:18.000000000 +0200
+++ gdal-1.4.2-fedora-patched/ogr/ogrsf_frmts/ogr_formats.html	2007-07-13 14:22:08.000000000 +0200
@@ -85,11 +85,6 @@
 </td><td> Yes
 </td></tr>
 
-<tr><td> <a href="drv_dgn.html">Microstation DGN</a>
-</td><td> Yes
-</td><td> No
-</td></tr>
-
 <tr><td> <a href="drv_mysql.html">MySQL</a>
 </td><td> No
 </td><td> No
diff -Nru gdal-1.4.2-fedora/ogr/ogrsf_frmts/ogrsf_frmts.h gdal-1.4.2-fedora-patched/ogr/ogrsf_frmts/ogrsf_frmts.h
--- gdal-1.4.2-fedora/ogr/ogrsf_frmts/ogrsf_frmts.h	2007-06-27 20:46:19.000000000 +0200
+++ gdal-1.4.2-fedora-patched/ogr/ogrsf_frmts/ogrsf_frmts.h	2007-07-13 14:22:08.000000000 +0200
@@ -310,7 +310,6 @@
 void CPL_DLL RegisterOGRPG();
 void CPL_DLL RegisterOGRMySQL();
 void CPL_DLL RegisterOGROCI();
-void CPL_DLL RegisterOGRDGN();
 void CPL_DLL RegisterOGRGML();
 void CPL_DLL RegisterOGRKML();
 void CPL_DLL RegisterOGRAVCBin();
@@ -321,8 +320,6 @@
 void CPL_DLL RegisterOGRDODS();
 void CPL_DLL RegisterOGRSQLite();
 void CPL_DLL RegisterOGRCSV();
-void CPL_DLL RegisterOGRILI1();
-void CPL_DLL RegisterOGRILI2();
 void CPL_DLL RegisterOGRGRASS();
 void CPL_DLL RegisterOGRPGeo();
 void CPL_DLL RegisterOGRDXFDWG();
--- gdal-1.4.2-fedora/gcore/gdaldefaultoverviews.cpp.orig	2007-06-27 20:47:49.000000000 +0200
+++ gdal-1.4.2-fedora-patched/gcore/gdaldefaultoverviews.cpp	2007-07-25 10:11:26.000000000 +0200
@@ -327,31 +327,6 @@
 /*      produce the imagery.                                            */
 /* -------------------------------------------------------------------- */
 
-#ifndef WIN32CE
-
-    if( bOvrIsAux )
-    {
-        eErr = HFAAuxBuildOverviews( osOvrFilename, poDS, &poODS,
-                                     nBands, panBandList,
-                                     nNewOverviews, panNewOverviewList, 
-                                     pszResampling, 
-                                     pfnProgress, pProgressData );
-
-        int j;
-        
-        for( j = 0; j < nOverviews; j++ )
-        {
-            if( panOverviewList[j] > 0 )
-                panOverviewList[j] *= -1;
-        }
-    }
-
-/* -------------------------------------------------------------------- */
-/*      Build new overviews - TIFF.  Close TIFF files while we          */
-/*      operate on it.                                                  */
-/* -------------------------------------------------------------------- */
-    else
-#endif /* WIN32CE */
     {
         if( poODS != NULL )
         {
