diff -rupN --no-dereference gdal-3.4.1-fedora/doc/Makefile gdal-3.4.1-fedora-new/doc/Makefile
--- gdal-3.4.1-fedora/doc/Makefile	2021-12-27 21:25:11.000000000 +0100
+++ gdal-3.4.1-fedora-new/doc/Makefile	2022-01-04 13:58:02.787551314 +0100
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    ?= --keep-going -j auto -W
+SPHINXOPTS    ?= --keep-going
 SPHINXBUILD   = sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
