--- gdal-2.0.0/swig/include/perl/ogr_perl.i	2015-06-14 21:06:33.000000000 +0200
+++ gdal-2.0.0-fedora/swig/include/perl/ogr_perl.i	2015-06-24 00:04:22.935551746 +0200
@@ -109,11 +109,11 @@
 
 }
 
-# wrapped data source methods:
+/* wrapped data source methods: */
 %rename (_GetDriver) GetDriver;
 %rename (_TestCapability) TestCapability;
 
-# wrapped layer methods:
+/* wrapped layer methods: */
 %rename (_ReleaseResultSet) ReleaseResultSet;
 %rename (_CreateLayer) CreateLayer;
 %rename (_DeleteLayer) DeleteLayer;
@@ -121,11 +121,11 @@
 %rename (_DeleteField) DeleteField;
 %rename (_Validate) Validate;
 
-# wrapped feature methods:
+/* wrapped feature methods: */
 %rename (_AlterFieldDefn) AlterFieldDefn;
 %rename (_SetGeometry) SetGeometry;
 
-# wrapped geometry methods:
+/* wrapped geometry methods: */
 %rename (_ExportToWkb) ExportToWkb;
 
 %perlcode %{
diff -up gdal-2.0.0-fedora/swig/include/perl/typemaps_perl.i.format gdal-2.0.0-fedora/swig/include/perl/typemaps_perl.i
--- gdal-2.0.0-fedora/swig/include/perl/typemaps_perl.i.format	2015-06-14 13:06:33.000000000 -0600
+++ gdal-2.0.0-fedora/swig/include/perl/typemaps_perl.i	2015-06-23 09:23:56.605781882 -0600
@@ -146,7 +146,7 @@
 {
  /* %typemap(ret) VSI_RETVAL */
   if ($1 == -1 ) {
-    croak(strerror(errno));
+    croak("%s",strerror(errno));
   }
 }
 
