diff -rupN --no-dereference gdal-3.2.1-fedora/doc/Makefile gdal-3.2.1-fedora-new/doc/Makefile
--- gdal-3.2.1-fedora/doc/Makefile	2020-12-29 13:15:46.000000000 +0100
+++ gdal-3.2.1-fedora-new/doc/Makefile	2021-01-04 21:49:36.218248119 +0100
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = --keep-going -j auto -W
+SPHINXOPTS    = --keep-going
 SPHINXBUILD   = sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
