diff -rup ./alg/thinplatespline.h gdal/alg/thinplatespline.h
--- ./alg/thinplatespline.h	2016-07-07 12:35:35.000000000 +0200
+++ gdal/alg/thinplatespline.h	2016-10-16 09:36:09.529176294 +0200
@@ -184,5 +184,5 @@ class VizGeorefSpline2D
     int *index; // [VIZ_GEOREF_SPLINE_MAX_POINTS];
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(VizGeorefSpline2D);
+    CPL_DISALLOW_COPY_ASSIGN(VizGeorefSpline2D)
 };
diff -rup ./frmts/envisat/adsrange.cpp gdal/frmts/envisat/adsrange.cpp
--- ./frmts/envisat/adsrange.cpp	2016-07-07 12:35:32.000000000 +0200
+++ gdal/frmts/envisat/adsrange.cpp	2016-10-16 09:36:09.709178641 +0200
@@ -74,7 +74,7 @@ class DataSet
     }
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(DataSet);
+    CPL_DISALLOW_COPY_ASSIGN(DataSet)
 } ;
 
 /* -------------------------------------------------------------------- */
diff -rup ./frmts/raw/rawdataset.h gdal/frmts/raw/rawdataset.h
--- ./frmts/raw/rawdataset.h	2016-07-07 12:35:34.000000000 +0200
+++ gdal/frmts/raw/rawdataset.h	2016-10-16 09:36:10.933194599 +0200
@@ -61,7 +61,7 @@ class CPL_DLL RawDataset : public GDALPa
                  ~RawDataset() = 0;
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(RawDataset);
+    CPL_DISALLOW_COPY_ASSIGN(RawDataset)
 };
 
 /************************************************************************/
@@ -173,7 +173,7 @@ public:
     int          GetOwnsFP() { return bOwnsFP; }
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(RawRasterBand);
+    CPL_DISALLOW_COPY_ASSIGN(RawRasterBand)
 };
 
 #endif // GDAL_FRMTS_RAW_RAWDATASET_H_INCLUDED
diff -rup ./gcore/gdaloverviewdataset.cpp gdal/gcore/gdaloverviewdataset.cpp
--- ./gcore/gdaloverviewdataset.cpp	2016-07-07 12:35:34.000000000 +0200
+++ gdal/gcore/gdaloverviewdataset.cpp	2016-10-16 09:36:05.805127741 +0200
@@ -94,7 +94,7 @@ class GDALOverviewDataset : public GDALD
         virtual int        CloseDependentDatasets();
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALOverviewDataset);
+    CPL_DISALLOW_COPY_ASSIGN(GDALOverviewDataset)
 };
 
 /* ******************************************************************** */
@@ -119,7 +119,7 @@ class GDALOverviewBand : public GDALProx
         virtual GDALRasterBand *GetOverview(int);
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALOverviewBand);
+    CPL_DISALLOW_COPY_ASSIGN(GDALOverviewBand)
 };
 
 /************************************************************************/
diff -rup ./gcore/gdal_pam.h gdal/gcore/gdal_pam.h
--- ./gcore/gdal_pam.h	2016-07-07 12:35:34.000000000 +0200
+++ gdal/gcore/gdal_pam.h	2016-10-16 09:36:05.765127219 +0200
@@ -183,7 +183,7 @@ class CPL_DLL GDALPamDataset : public GD
     void   SetPamFlags(int nValue ) { nPamFlags = nValue; }
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALPamDataset);
+    CPL_DISALLOW_COPY_ASSIGN(GDALPamDataset)
 };
 
 /* ==================================================================== */
@@ -298,7 +298,7 @@ class CPL_DLL GDALPamRasterBand : public
     GDALRasterBandPamInfo *GetPamInfo() { return psPam; }
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALPamRasterBand);
+    CPL_DISALLOW_COPY_ASSIGN(GDALPamRasterBand)
 };
 
 // These are mainly helper functions for internal use.
diff -rup ./gcore/gdal_priv.h gdal/gcore/gdal_priv.h
--- ./gcore/gdal_priv.h	2016-07-07 12:35:35.000000000 +0200
+++ gdal/gcore/gdal_priv.h	2016-10-16 09:36:05.637125551 +0200
@@ -107,7 +107,7 @@ public:
     void        Clear();
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALMultiDomainMetadata);
+    CPL_DISALLOW_COPY_ASSIGN(GDALMultiDomainMetadata)
 };
 
 /* ******************************************************************** */
@@ -226,7 +226,7 @@ class CPL_DLL GDALDefaultOverviews
     char**     GetSiblingFiles() { return papszInitSiblingFiles; }
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALDefaultOverviews);
+    CPL_DISALLOW_COPY_ASSIGN(GDALDefaultOverviews)
 };
 
 /* ******************************************************************** */
@@ -266,7 +266,7 @@ class CPL_DLL GDALOpenInfo
     bool        AreSiblingFilesLoaded() const;
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALOpenInfo);
+    CPL_DISALLOW_COPY_ASSIGN(GDALOpenInfo)
 };
 
 /* ******************************************************************** */
@@ -560,7 +560,7 @@ private:
     OGRStyleTable      *m_poStyleTable;
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALDataset);
+    CPL_DISALLOW_COPY_ASSIGN(GDALDataset)
 };
 
 /* ******************************************************************** */
@@ -647,7 +647,7 @@ class CPL_DLL GDALRasterBlock
     static void DestroyRBMutex();
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALRasterBlock);
+    CPL_DISALLOW_COPY_ASSIGN(GDALRasterBlock)
 };
 
 /* ******************************************************************** */
@@ -920,7 +920,7 @@ class CPL_DLL GDALRasterBand : public GD
     void ReportError(CPLErr eErrClass, CPLErrorNum err_no, const char *fmt, ...)  CPL_PRINT_FUNC_FORMAT (4, 5);
 
 private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALRasterBand);
+    CPL_DISALLOW_COPY_ASSIGN(GDALRasterBand)
 };
 
 /* ******************************************************************** */
@@ -1121,7 +1121,7 @@ class CPL_DLL GDALDriver : public GDALMa
     CPLErr              DefaultCopyFiles( const char * pszNewName,
                                           const char * pszOldName );
 private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALDriver);
+    CPL_DISALLOW_COPY_ASSIGN(GDALDriver)
 };
 
 /* ******************************************************************** */
diff -rup ./gcore/gdal_proxy.h gdal/gcore/gdal_proxy.h
--- ./gcore/gdal_proxy.h	2016-07-07 12:35:35.000000000 +0200
+++ gdal/gcore/gdal_proxy.h	2016-10-16 09:36:05.681126124 +0200
@@ -94,7 +94,7 @@ class CPL_DLL GDALProxyDataset : public
         virtual CPLErr          CreateMaskBand( int nFlags );
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALProxyDataset);
+    CPL_DISALLOW_COPY_ASSIGN(GDALProxyDataset)
 };
 
 /* ******************************************************************** */
@@ -193,7 +193,7 @@ class CPL_DLL GDALProxyRasterBand : publ
                                                 GIntBig *pnLineSpace,
                                                 char **papszOptions );
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALProxyRasterBand);
+    CPL_DISALLOW_COPY_ASSIGN(GDALProxyRasterBand)
 };
 
 
@@ -257,7 +257,7 @@ class CPL_DLL GDALProxyPoolDataset : pub
         virtual const char *GetGCPProjection();
         virtual const GDAL_GCP *GetGCPs();
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALProxyPoolDataset);
+    CPL_DISALLOW_COPY_ASSIGN(GDALProxyPoolDataset)
 };
 
 /* ******************************************************************** */
@@ -312,7 +312,7 @@ class CPL_DLL GDALProxyPoolRasterBand :
         virtual GDALRasterBand *GetRasterSampleOverview( GUIntBig nDesiredSamples); // TODO
         virtual GDALRasterBand *GetMaskBand();
   private:
-    CPL_DISALLOW_COPY_ASSIGN(GDALProxyPoolRasterBand);
+    CPL_DISALLOW_COPY_ASSIGN(GDALProxyPoolRasterBand)
 };
 
 /* ******************************************************************** */
diff -rup ./ogr/ogr_feature.h gdal/ogr/ogr_feature.h
--- ./ogr/ogr_feature.h	2016-07-07 12:35:28.000000000 +0200
+++ gdal/ogr/ogr_feature.h	2016-10-16 09:36:05.965129827 +0200
@@ -119,7 +119,7 @@ class CPL_DLL OGRFieldDefn
     int                 IsSame( const OGRFieldDefn * ) const;
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(OGRFieldDefn);
+    CPL_DISALLOW_COPY_ASSIGN(OGRFieldDefn)
 };
 
 /************************************************************************/
@@ -176,7 +176,7 @@ public:
         int                 IsSame( OGRGeomFieldDefn * );
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(OGRGeomFieldDefn);
+    CPL_DISALLOW_COPY_ASSIGN(OGRGeomFieldDefn)
 };
 
 /************************************************************************/
@@ -260,7 +260,7 @@ class CPL_DLL OGRFeatureDefn
     static void         DestroyFeatureDefn( OGRFeatureDefn * );
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(OGRFeatureDefn);
+    CPL_DISALLOW_COPY_ASSIGN(OGRFeatureDefn)
 };
 
 /************************************************************************/
@@ -447,7 +447,7 @@ class CPL_DLL OGRFeature
     static void         DestroyFeature( OGRFeature * );
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(OGRFeature);
+    CPL_DISALLOW_COPY_ASSIGN(OGRFeature)
 };
 
 /************************************************************************/
diff -rup ./ogr/ogrsf_frmts/ili/imdreader.cpp gdal/ogr/ogrsf_frmts/ili/imdreader.cpp
--- ./ogr/ogrsf_frmts/ili/imdreader.cpp	2016-07-07 12:35:31.000000000 +0200
+++ gdal/ogr/ogrsf_frmts/ili/imdreader.cpp	2016-10-16 09:36:07.009143439 +0200
@@ -334,7 +334,7 @@ public:
     }
 
   private:
-    CPL_DISALLOW_COPY_ASSIGN(IliClass);
+    CPL_DISALLOW_COPY_ASSIGN(IliClass)
 };
 
 
