diff -rupN --no-dereference gdal-3.1.4-fedora/doc/Makefile gdal-3.1.4-fedora-new/doc/Makefile
--- gdal-3.1.4-fedora/doc/Makefile	2020-10-20 11:47:08.000000000 +0200
+++ gdal-3.1.4-fedora-new/doc/Makefile	2020-11-02 11:56:49.173328561 +0100
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = --keep-going -j auto -W
+SPHINXOPTS    = --keep-going
 SPHINXBUILD   = sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
