diff -up gdal-1.5.2-fedora/frmts/dods/dodsdataset2.cpp.libdap~ gdal-1.5.2-fedora/frmts/dods/dodsdataset2.cpp
--- gdal-1.5.2-fedora/frmts/dods/dodsdataset2.cpp.libdap~	2008-09-09 16:49:39.000000000 +0200
+++ gdal-1.5.2-fedora/frmts/dods/dodsdataset2.cpp	2008-09-09 16:50:05.000000000 +0200
@@ -67,6 +67,8 @@ CPL_C_START
 void GDALRegister_DODS(void);
 CPL_C_END
 
+using namespace libdap ;
+
 /** Attribute names used to encode geo-referencing information. Note that
     these are not C++ objects to avoid problems with static global
     constructors. 
diff -up gdal-1.5.2-fedora/ogr/ogrsf_frmts/dods/ogr_dods.h.libdap~ gdal-1.5.2-fedora/ogr/ogrsf_frmts/dods/ogr_dods.h
--- gdal-1.5.2-fedora/ogr/ogrsf_frmts/dods/ogr_dods.h.libdap~	2008-09-09 16:50:50.000000000 +0200
+++ gdal-1.5.2-fedora/ogr/ogrsf_frmts/dods/ogr_dods.h	2008-09-09 16:51:19.000000000 +0200
@@ -68,6 +68,8 @@
 #include <Error.h>
 #include <escaping.h>
 
+using namespace libdap ;
+
 /************************************************************************/
 /*                           OGRDODSFieldDefn                           */
 /************************************************************************/
