--- frmts/ilwis/ilwisdataset.cpp.orig
+++ frmts/ilwis/ilwisdataset.cpp
@@ -407,5 +407,4 @@
 {
     string st = ReadElement("MapStore", "Type", pszFileName.c_str());
-    transform(st.begin(), st.end(), st.begin(), tolower);
 		
     if( EQUAL(st.c_str(),"byte"))
@@ -482,6 +481,4 @@
     else
         georef = ReadElement("MapList", "GeoRef", osFileName);
-
-    transform(georef.begin(), georef.end(), georef.begin(), tolower);
 
     //Capture the geotransform, only if the georef is not 'none', 
@@ -789,7 +786,6 @@
         //	Fetch coordinate system
         string csy = ReadElement("GeoRef", "CoordSystem", pszGeoRef);
-        transform(csy.begin(), csy.end(), csy.begin(), tolower);
-
         string pszProj;
+
         if( (csy.length() != 0) && !EQUAL(csy.c_str(),"unknown.csy"))
         {
@@ -797,5 +793,5 @@
             //Form the coordinate system file name
             if( !(EQUALN( csy.c_str(), "latlon.csy", 10 )) && 
-                !(EQUALN( csy.c_str(), "LatlonWGS84.csy", 15 )))			
+                !(EQUALN( csy.c_str(), "LatlonWGS84.csy", 15 )))            
             {
                 string pszBaseName = string(CPLGetBasename(csy.c_str()) );
@@ -1319,5 +1315,4 @@
     string domName = ReadElement("BaseMap", "Domain", pszFileName.c_str());
     string pszBaseName = string(CPLGetBasename( domName.c_str() ));
-    transform(pszBaseName.begin(), pszBaseName.end(), pszBaseName.begin(), tolower);
     string pszPath = string(CPLGetPath( pszFileName.c_str() ));
 		
@@ -1371,5 +1366,4 @@
 				
         string domType = ReadElement("Domain", "Type", pszDomainName.c_str());
-        transform(domType.begin(), domType.end(), domType.begin(), tolower);
         if EQUAL(domType.c_str(),"domainvalue")  
             {
--- frmts/idrisi/IdrisiDataset.cpp.orig	2008-02-13 22:21:36.000000000 +0200
+++ frmts/idrisi/IdrisiDataset.cpp	2008-02-13 22:22:27.000000000 +0200
@@ -35,6 +35,7 @@
 #include "gdal_pam.h"
 #include "gdal_rat.h"
 #include "gdal_alg.h"
+#include <climits>
 
 CPL_CVSID( "$Id: IdrisiDataset.cpp 12896 2007-11-20 21:01:46Z rouault $" );
 
--- ogr/ogrsf_frmts/geojson/ogrgeojsonlayer.cpp.orig	2008-02-14 02:43:16.000000000 +0200
+++ ogr/ogrsf_frmts/geojson/ogrgeojsonlayer.cpp	2008-02-14 13:21:22.000000000 +0200
@@ -26,10 +26,11 @@
  * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
  * DEALINGS IN THE SOFTWARE.
  ****************************************************************************/
+#include <algorithm> // for_each, find_if
+
 #include "ogr_geojson.h"
 #include "ogrgeojsonwriter.h"
 #include <jsonc/json.h> // JSON-C
-#include <algorithm> // for_each, find_if
 
 /************************************************************************/
 /*                       STATIC MEMBERS DEFINITION                      */
