--- swig/perl/gdal_wrap.cpp.orig	2008-03-25 20:19:03.000000000 +0200
+++ swig/perl/gdal_wrap.cpp	2008-03-25 20:19:52.000000000 +0200
@@ -1228,8 +1228,11 @@
 /* Macros for low-level exception handling */
 #define SWIG_croak(x)    { SWIG_Error(SWIG_RuntimeError, x); SWIG_fail; }
 
+#ifndef XSPROTO
+# define XSPROTO(name) void name(pTHX_ CV* cv)
+#endif 
 
-typedef XS(SwigPerlWrapper);
+typedef XSPROTO(SwigPerlWrapper);
 typedef SwigPerlWrapper *SwigPerlWrapperPtr;
 
 /* Structure for command table */
--- swig/perl/ogr_wrap.cpp.orig	2008-03-25 20:45:42.000000000 +0200
+++ swig/perl/ogr_wrap.cpp	2008-03-25 20:48:00.000000000 +0200
@@ -1228,8 +1228,11 @@
 /* Macros for low-level exception handling */
 #define SWIG_croak(x)    { SWIG_Error(SWIG_RuntimeError, x); SWIG_fail; }
 
+#ifndef XSPROTO
+# define XSPROTO(name) void name(pTHX_ CV* cv)
+#endif
 
-typedef XS(SwigPerlWrapper);
+typedef XSPROTO(SwigPerlWrapper);
 typedef SwigPerlWrapper *SwigPerlWrapperPtr;
 
 /* Structure for command table */
--- swig/perl/osr_wrap.cpp.orig	2008-03-25 20:49:16.000000000 +0200
+++ swig/perl/osr_wrap.cpp	2008-03-25 20:49:33.000000000 +0200
@@ -1229,7 +1229,11 @@
 #define SWIG_croak(x)    { SWIG_Error(SWIG_RuntimeError, x); SWIG_fail; }
 
 
-typedef XS(SwigPerlWrapper);
+#ifndef XSPROTO
+# define XSPROTO(name) void name(pTHX_ CV* cv)
+#endif
+
+typedef XSPROTO(SwigPerlWrapper);
 typedef SwigPerlWrapper *SwigPerlWrapperPtr;
 
 /* Structure for command table */
