diff -rupN --no-dereference gdal-3.2.2-fedora/doc/Makefile gdal-3.2.2-fedora-new/doc/Makefile
--- gdal-3.2.2-fedora/doc/Makefile	2021-03-05 12:38:35.000000000 +0100
+++ gdal-3.2.2-fedora-new/doc/Makefile	2021-03-24 13:44:40.886428785 +0100
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = --keep-going -j auto -W
+SPHINXOPTS    = --keep-going
 SPHINXBUILD   = sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
